modules/label_logic.rb

```diff
@@ -0,0 +1,50 @@
+require_relative '../app'
+
+class LabelLogic
+  def initialize
+    @all_local_labels = load_labels
+  end
+
+  def add_label
+    print 'Enter label (e.g Gift, New): '
+    input_label = gets.chomp
+    create_book_label(input_label)
+  end
```

Comment:
- [REQUIRED] When adding the label, the user should be asked for the colour. Currently, there is no input for the colour. 💻 

- [OPTIONAL] I would advise that you use _**'Enter Title'**_ instead of `'Enter label (e.g Gift, New): '`. This makes it easier for the user to know what needs to be inputted. 👍

Reply:
Hello @Gambit142 👋️, thank you for the feedback. We will implement the changes🤗️